notebooks/figures.ipynb

```diff
@@ -0,0 +1,1755 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/15317110/205666538-75f36817-bba9-4144-99ea-6a7b8b319e83.png)

I initially picked immunochem/anatomy, nanotech/microscopy, and proteomics/metabolomics as examples since they have different correlations. Then I threw in comp bio/genetics because I understand the fields and can better discuss the papers that were highlighted